### ISSUE TITLE
PDFs saved with non-disclosed bookmark hierarchies do not include the undisclosed sections

### ISSUE THREAD

**Issue by kdv24** [react-pdf-viewer/react-pdf-viewer#1441]
If a pdf is created that contains bookmarks, if it is saved with the bookmarks all closed (undisclosed), only the single top level bookmark is in the hierarchy in the pdf-viewer. There are no lower levels to disclose. We can solve this on our end by making sure PDFs are saved with all levels open, but it seems like a bug to fix on the react-pdf-viewer side.

**Comment by phuocng** [comment]
@kdv24 Can you let me know the steps (maybe including the sample document) to reproduce the issue?

**Comment by kdv24** [comment]
Hi @phuocng. I'm unable to include the sample document, but the steps to reproduce are as follows:

1. Create a pdf with bookmarks that each open additional bookmarks. We have bookmarks that go as much as five levels deep, meaning that a bookmark can be clicked on and will then display multiple nested bookmarks that can each also open additional nested bookmarks.
2. When you make the pdf, save it with all the bookmarks closed so only the top level bookmark is visible.
3. Open the pdf in react-pdf-viewer and it only displays that top level bookmark and none of the other levels are included.  
4. If the pdf is saved with the bookmarks open, the other levels of bookmarks are visible.

Does that make sense? If not, I can try to explain further.  

Thanks for looking!

**Comment by kdv24** [comment]
@phuocng were you able to replicate the issue I described? Thanks!

**Comment by phuocng** [comment]
Not yet. When you mention _Create a pdf with bookmarks_, does this mean that you use an external tool (such as Adobe Acrobat Pro) to create a PDF document?

**Comment by kdv24** [comment]
Yes, Adobe Acrobat Pro. On Mar 6, 2023, at 5:55 AM, phuocng ***@***.***> wrote:﻿
Not yet. When you mention Create a pdf with bookmarks, does this mean that you use an external tool (such as Adobe Acrobat Pro) to create a PDF document?

**Comment by phuocng** [comment]
I don't have that tool installed. Can you create and give me a sample document with that tool?

**Comment by kdv24** [comment]
@phuocng I have a couple of sample documents I can share with you privately. What would be the best way for me to do so?

**Comment by phuocng** [comment]
@kdv24 Can you please share to me@phuoc.ng? I will ensure that the document is used for testing only.
